doc/Overview/Mvux/States.md

```diff
@@ -2,45 +2,47 @@
 
-Contrary to Feeds, States are stateful (hence the name!), and do keep track of the state of the Model and its entities.  
+Contrary to Feeds, states are stateful (as the name suggests) in that they keep a record of the current data value. States also permit the current value to be modified, which is useful for two-way binding scenarios.
 
 MVUX utilizes its powerful code-generation engine to generate a bindable proxy for each Model, which holds the state information of the data, as well as a bindable proxy for entities where needed, for instance, if the entities are immutable (e.g. records - the recommended type).  
-The bindable proxies use as a bridge that enables immutable entities to work with the WinUI data-binding engine. The States in the Model are monitored for data-binding changes, and in response to any change, the objects are recreated fresh, instead of their properties being changed.
+The bindable proxies use as a bridge that enables immutable entities to work with the WinUI data-binding engine. The states in the Model are monitored for data-binding changes, and in response to any change, the objects are recreated fresh, instead of their properties being changed.
 
-> [!NOTE]
-> States keep the state of the data, so every new subscription to them, (such as awaiting them or binding them to an additional control, etc.), will use the data currently loaded in the State (if any).  
-To reload the data, a refresh is required.
+States keep the current value of the data, so every new subscription to them, (such as awaiting them or binding them to an additional control, etc.), will use the data currently loaded in the state (if any).  
+
+Like a feed, states can be reloaded, which will invoke the asynchronous operation that is used to create the state.
 
 States and Feeds are different in the following:
 
 1. When subscribing to a state, the currently loaded value is going to be replayed.
-2. A State provides the `Update` method that allows changing its current value.
+2. A state provides the `Update` method that allows changing its current value.
 3. States are attached to an owner and share the same lifetime as that owner.
-4. The main usage of a State is for two-way bindings.
+4. The main usage of a state is for two-way bindings.
 
 ## States are attached to their owner
 
-Besides holding the state information, a reference to the bindable proxy is shared with the States so that when the View is closed and disposed of, it tunnels down to the States and the Models and makes them available for garbage collection. It shares the same lifetime as its owner.
+Besides holding the state information, a reference to the Model is shared with the states so that when the View is closed and disposed of, it tunnels down to the states and the Models and makes them available for garbage collection. States share the same lifetime as its owner.
```

Comment:
```suggestion
Besides holding the state information, a reference to the Model is shared with the states so that when the View is closed and disposed of, it tunnels down to the states and the Models and makes them available for garbage collection. States share the same lifetime as their owner.
```